butte-build/src/codegen.rs

```diff
@@ -787,6 +913,11 @@ impl<'a> CodeGenerator<'a> {
 
     pub fn build_tokens(&mut self, tokens: &mut TokenStream) {
         let mut rpc_gen = self.rpc_gen.take();
+
+        (quote! {
+            #[allow(clippy::type_complexity)]
```

Comment:
:grimacing:

Reply:
We can't do much about that one right, since we generate pretty complex types with like `Option<WIPOffset<Vector<'a, SomeTable<...>>`.  With this though, the generated modules pass Clippy's default lints.

Reply:
Yeah, totally :)